kedro_neptune/hooks.py

```diff
@@ -35,6 +38,20 @@ def log_dataset_metadata(run: Run, base_namespace: str, name: str, dataset: Abst
         'version': dataset._version if hasattr(dataset, '_version') else None
     }
 
+    if isinstance(dataset, AbstractNeptuneDataSet):
```

Comment:
In addition, we should check for the existence of the dataset

Reply:
Why?

`isinstance(None, int)` returns `False`

Reply:
Not such existance but dome datasets are created during runtime. I mean https://kedro.readthedocs.io/en/stable/kedro.io.AbstractDataSet.html#kedro.io.AbstractDataSet.exists